src/YearnStrategy.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.19;
+
+import {ERC20} from "solady/src/tokens/ERC20.sol";
+import {SafeTransferLib} from "solady/src/utils/SafeTransferLib.sol";
+
+import {IYearnGauge} from "src/interfaces/IYearnGauge.sol";
+import {ISDTDistributor} from "src/interfaces/ISDTDistributor.sol";
+import {IYearnRewardPool} from "src/interfaces/IYearnRewardPool.sol";
+import {ILocker, SafeExecute, Strategy, ILiquidityGauge, UUPSUpgradeable} from "src/strategy/StrategyV1.sol";
+
+/// @notice Main access point of Yearn Locker.
+contract YearnStrategy is Strategy {
+    using SafeExecute for ILocker;
+
+    /// @notice Reward Pool Contract to distribute DYFI.
+    /// @dev Need to set it at deployment since it's a proxy.
+    address public dyfiRewardPool = 0x2391Fc8f5E417526338F5aa3968b1851C16D894E;
```

Comment:
Using the address book would be great. I've been tracking flying addresses since I started considering the next version of the address book 😁